main.cpp

```diff
@@ -52,24 +52,30 @@ void printword(const string word, const vector<char> cg) // function to print
 
 int main()
 {
+    bool isGameRunning{true};                                                                              // turns the game on or off
+    const int maxguesses{10};                                                                              // max number of guess before the user loses
     char input;                                                                                            // for user's letter guesses
     int choice;                                                                                            // for user choice
     string wordinput;                                                                                      // for when user wants to guess the entire word
     vector<char> guessedchar;                                                                              // array for all the letters the user have guessed
     vector<string> list{"special", "random", "roast", "firetruck", "throw", "trend", "blast", "gasoline"}; // list of possible questions
     vector<char> correctguesses;                                                                           // letters that were guessed correctly
-    int guesses;                                                                                           // number of guesses by the user
+    int guesses{0};                                                                                        // number of guesses by the user
     srand(time(NULL));
     int random = rand() % list.size(); // randomise the word
     string word = list[random];
     printoptions(); // to print the options
     // Start Code
-    while (guesses <= 10)
+    while (isGameRunning == false) // when game is not running
     {
-        cout << "Remaining Guesses: " << 10 - guesses << endl;
+        return 0;
+    }
+    while (guesses <= maxguesses && isGameRunning == true) // for when game is running
```

Comment:
this is ok, but i think it's better to simply check for 'isGameRunning'. Then, within this loop, whenever you increase the 'guesses' variable, you can do
`if(guesses >= maxguesses) isGrameRunning = false`

causing the main loop to stop